app/controllers/events/list.js

```diff
@@ -73,13 +73,13 @@ export default Controller.extend({
         event.destroyRecord();
       })
         .then(() => {
+          this.set('isLoading', false);
           this.notify.success(this.get('l10n').t('Event has been deleted successfully.'));
+          this.send('refreshRoute');
         })
         .catch(() => {
-          this.notify.error(this.get('l10n').t('An unexpected error has occurred.'));
-        })
-        .finally(() => {
```

Comment:
why did you remove the `finally` block.

Reply:
Since the due to the  'finally' block 'An unexpected block' was getting executed again ,So when you will delete an event 'Event has been deleted successfully' And 'An unexpected error' Both of them gets appear on tooltip. So to avoid that i removed the finally handler of promise.

Reply:
finally block needs to be intact, and should be responsible for setting `isLoading` to `false`. `isLoading` does not need to be set `false` separately for success and failure cases.

Reply:
I did the changes , Looks fine now?